### PR TITLE
Implement compare command and fix a print statement bug

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: swapper/swapper.py [hunk 1]
```diff
@@ -106,3 +108,101 @@ def copy_and_replace(fc):
...
         except:
             print(f'could not update privileges to {renamed_fc_sgid10}')
+
+
+def compare():
+    '''compares data sets between SGID and SGID10 and returns the tables that are different
+    '''
+    dbo_owner = os.path.join(os.getenv('SWAPPER_CONNECTION_FILE_PATH'), 'SGID10', 'SGID10_sde.sde')
+
+    if not os.path.exists(dbo_owner):
```
Comment (steveoh) [inline swapper/swapper.py]: i would recommend replacing all os.path usages with pathlib in any new code you write.

File: swapper/swapper.py [hunk 2]
```diff
@@ -106,3 +108,101 @@ def copy_and_replace(fc):
...
+        print(f'{dbo_owner} does not exist')
+
+        return []
+
+    tables_needing_update = []
+
+    internal_connection = pyodbc.connect('DRIVER={SQL Server};SERVER=internal.agrc.utah.gov;DATABASE=SGID;UID=internal;PWD=internal')
```
Comment (steveoh) [inline swapper/swapper.py]: this probably should probably be removed...

File: swapper/swapper.py [hunk 3]
```diff
@@ -106,3 +108,101 @@ def copy_and_replace(fc):
...
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name IN ({join_strings(tables)}) AND LOWER(column_name) NOT IN ({join_strings(skip_fields)})'''
+    field_meta = cursor.execute(field_meta_query).fetchall()
+
+    table_field_map = {}
+
+    for db, schema, table, field, field_type in field_meta:
+        full_table_name = f'{db}.{schema}.{table}'
+        if field_type == 'geometry':
+            field = f'{field}.STAsText() as {field}'
```
Comment (steveoh) [inline swapper/swapper.py:196]: I think this is going to cause issues with precision but maybe not?

**Review by steveoh — CHANGES_REQUESTED** [review]

File: swapper/swapper.py (hunk 3 above)
Comment (stdavis) [reply swapper/swapper.py:196]: Not in my testing.

Comment (stdavis) [reply swapper/swapper.py:196]: Did you have issues with it in Cambiador? I assume that you're worried about false positives on changes?

File: swapper/swapper.py (hunk 2 above)
Comment (stdavis) [reply swapper/swapper.py]: Done.

File: swapper/swapper.py (hunk 1 above)
Comment (stdavis) [reply swapper/swapper.py]: Done.

**Review by steveoh — APPROVED** [review]

**Comment by stdavis** [comment]
Just removed some test code to limit the number of tables selected. Thanks for the view, @steveoh !

**Pull request merged**